### DISCUSSION TITLE
論Python sorted()

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#155] [General]
sorted()的功用：把資料型態先變成list，經過升冪排序，產生並return產生的list
疑問：return就是輸出嗎？
1.輸入list(串列)作為sorted()的引數(argument)，給sorted的參數(parameter)使用
在jupyter lab輸入
`sorted([7,8,1,34,18,8])`
並按ctrl+enter執行
會得到
`[1, 7, 8, 8, 18, 34]`
![image](https://user-images.githubusercontent.com/86505454/141676165-5e0e18c0-ea6d-4feb-94e9-5f941d64d774.png)

**Comment by chia-kai-chang** [comment]
謝謝 @enoch-lightning  的整理，另外`return`是function的輸出回傳(給call此function的對象，如：主程式)

**Reply by enoch-lightning** [reply]
原來return不是輸出，而是將「function=函數=函式」輸出的結果，傳給「呼叫=call」這個function的人。
